package.json

```diff
@@ -99,6 +99,7 @@
     "request": "2.87.0",
     "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git",
     "source-map-support": "^0.5.6",
+    "sprintf-js": "^1.1.1",
```

Comment:
We're not using any of the advanced formatting that this library gives us. Pretty much everything we're logging can be handed with Javascript template literals. We should be careful about adding dependencies, since eventually we're going to need to audit all of these.

What do you think about switching to JS template literals throughout the app?

Reply:
I believe js template literals are useful in most cases.
The purpose of this is to delay formatting until level check. So if you have:
`log.debug('debug message %j', superHugeObject)`
and your log level is `warn`, the message will not be formatted, and `superHugeObject` will not have to be serialized as opposed to:
``log.debug(`debug message ${JSON.stringify(superHugeObject)}`);``